proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/base/ProxyNodeAppRuleTestBase.java

```diff
@@ -94,21 +96,46 @@
             ConfigOverride.config("hubFacingEncryptionKeyPair.privateKey.key", TEST_PRIVATE_KEY)
     );
 
-    protected Response postEidasAuthnRequest(AuthnRequest eidasAuthnRequest) throws URISyntaxException {
+    protected Response postEidasAuthnRequest(AuthnRequest eidasAuthnRequest) {
+        System.out.println(marshaller.transformToString(eidasAuthnRequest));
         String encodedRequest = Base64.encodeAsString(marshaller.transformToString(eidasAuthnRequest));
         Form postForm = new Form().param(SamlFormMessageType.SAML_REQUEST, encodedRequest);
 
-        return proxyNodeAppRule.target("/SAML2/SSO/POST")
-            .request()
-            .post(Entity.form(postForm));
+        try {
+            return proxyNodeAppRule.target("/SAML2/SSO/POST")
+                    .request()
+                    .post(Entity.form(postForm));
+        } catch (URISyntaxException e) {
+            fail(e);
+            return null;
+        }
+    }
+
+    protected Response redirectEidasAuthnRequest(AuthnRequest eidasAuthnRequest) {
+        String encodedRequest = Base64.encodeAsString(marshaller.transformToString(eidasAuthnRequest));
+
+        try {
+            return proxyNodeAppRule.target("/SAML2/SSO/Redirect")
```

Comment:
What is the redirect endpoint used for?

Reply:
I think it's just another way of accessing a SAML journey. It allows you to use a GET rather than a POST? We don't support it for Verify but we have to for eIDAS.